File: app/src/main/java/com/kadek/t_1/Splash.java
package com.kadek.t_1;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class Splash extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);

        Thread splash = new Thread(){
            @Override
            public void run() {
                try{
                super.run();
                sleep(5000);
                } catch (Exception e){

                }finally {
                    Intent i = new Intent(Splash.this,MenuFront.class);
                    startActivity(i);
                    finish();
                }
            }
        };
        splash.start();
    }
}

File: app/src/main/java/com/kadek/t_1/Splash.kt
package com.kadek.t_1

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class Splash : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)

        val splash = object : Thread() {
            override fun run() {
                try {
                    super.run()
                    Thread.sleep(5000)
                } catch (e: Exception) {

                } finally {
                    val i = Intent(this@Splash, MenuFront::class.java)
                    startActivity(i)
                    finish()
                }
            }
        }
        splash.start()
    }
}
